discovery/marathon/marathon_test.go

```diff
@@ -358,6 +358,7 @@ func marathonTestAppListWithPortDefinitions(labels map[string]string, runningTas
 				{Labels: make(map[string]string), Port: 31000},
 				{Labels: labels, Port: 32000},
 			},
+			RequirePorts: true,
```

Comment:
I removed it. But test failed. I think that I need to add `Ports: []uint32{31000, 32000},` to a `task`.

Reply:
Correct, but I wouldn't use those ports, since they already correspond to the expected test results. I've pushed a commit here: https://github.com/prometheus/prometheus/commit/4a4a442def69f18c81272e72a92d8af81a53cab2